spyder/plugins/help/widgets.py

```diff
@@ -157,6 +159,8 @@ def __init__(self, parent):
         else:
             QWidget.__init__(self, parent)
             SpyderWidgetMixin.__init__(self, class_parent=parent)
+        from qtpy.QtWebEngineWidgets import QWebEnginePage
```

Comment:
```suggestion

        from qtpy.QtWebEngineWidgets import QWebEnginePage
```

To improve code clarity a bit.